ts-starter-complete-cotta/package.json

```diff
@@ -5,21 +5,39 @@
   "description": "Minecraft TypeScript Starter Project",
   "private": true,
   "devDependencies": {
-    "del": "^6.0.0",
-    "gulp": "^4.0.2",
-    "gulp-cli": "^2.3.0",
-    "gulp-sourcemaps": "^3.0.0",
-    "gulp-typescript": "^6.0.0-alpha.1",
+    "@minecraft/core-build-tasks": "./resources/minecraft-core-build-tasks-1.0.1-patch-30.tgz",
+    "eslint-plugin-minecraft-linting": "^1.0.1",
+    "npm-watch": "^0.11.0",
     "source-map": "^0.7.4",
-    "typescript": "^4.4.3"
+    "ts-node": "^10.9.1",
+    "typescript": "^5.0.2"
   },
   "scripts": {
+    "lint": "just-scripts lint",
+    "build": "just-scripts build",
+    "clean": "just-scripts clean",
+    "package": "just-scripts package",
+    "local-deploy": "just-scripts local-deploy",
+    "mcaddon": "just-scripts mcaddon",
+    "watch": "npm-watch",
     "enablemcloopback": "CheckNetIsolation.exe LoopbackExempt -a -p=S-1-15-2-1958404141-86561845-1752920682-3514627264-368642714-62675701-733520436",
     "enablemcpreviewloopback": "CheckNetIsolation.exe LoopbackExempt -a -p=S-1-15-2-424268864-5579737-879501358-346833251-474568803-887069379-4040235476"
   },
+  "watch": {
+    "local-deploy": {
+      "patterns": [
+        "scripts",
+        "behavior_packs",
+        "resource_packs"
+      ],
+      "quiet": false,
+      "extensions": "ts,json,lang,png"
+    }
+  },
   "dependencies": {
-    "@minecraft/server": "^1.5.0",
+    "@minecraft/math": "^1.1.0",
+    "@minecraft/server": "^1.8.0",
     "@minecraft/server-ui": "^1.1.0",
-    "decode-uri-component": "^0.2.2"
+    "@minecraft/vanilla-data": "^1.20.60"
```

Comment:
Is there anywhere you can use vanilla-data to make sure this is all good E2E (and the sample documentation can also be updated to reference this?)

Reply:
Same with @minecraft/math. Unless this will be addressed later?

Reply:
For the moment updated in some places across all the projects. But, I think we should do later to avoid mistakes in this huge PR.